code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -196,7 +196,9 @@
 		if(!xeno.counts_for_slots)
 			hive_xenos -= xeno
 
-	if(length(hive_xenos) > (length(GLOB.alive_human_list) * ENDGAME_LARVA_CAP_MULTIPLIER))
+	var/real_total_xeno_count = length(hive_xenos) + linked_hive.stored_larva
+
+	if(real_total_xeno_count > (length(GLOB.alive_human_list) * ENDGAME_LARVA_CAP_MULTIPLIER))
 		return
 
 	linked_hive.partial_larva += length(hive_xenos) * LARVA_ADDITION_MULTIPLIER
```

Comment:
This should probably use real_total_xeno_count too, considering we did just calculate it.

Reply:
True!